Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM colucom/nodejs:8.12.0
```

Comment:
should be fuseio

Reply:
will do this latter. colucom image is open to every one so we can use this for now